api_lib/api_lib.py

```
@@ -119,9 +90,27 @@ async def __call_api_with_timeout(
-            else:
-                return result
+            except OSError as err:
+                logger.error(
+                    "OSError while doing an API call",
+                    {
+                        "error": str(err),
+                        "host": self.host,
+                        "method": method.value,
+                        "path": path,
+                    },
+                )
```

Pylint (3.3.7)

[error] Too many arguments for logging format string

(E1205)